### ISSUE TITLE
ubuntu dependencies

### ISSUE THREAD

**Issue by pirate486743186** [trizen/pipe-viewer#7]
here are the ubuntu dependencies, so that others don't go dig for them. A simple copy paste should do. Not fully tested

pipe-viewer
`libwww-perl liblwp-protocol-https-perl libdata-dump-perl libjson-perl`

build module
`libmodule-build-perl`

gtk-pipe-viewer
`libgtk3-perl libfile-sharedir-perl`

optional
`liblwp-useragent-chicaching-perl libterm-readline-gnu-perl libjson-xs-perl`
`libunicode-linebreak-perl` or `libtext-charwidth-perl`

all
`libwww-perl liblwp-protocol-https-perl libdata-dump-perl libjson-perl libmodule-build-perl libgtk3-perl libfile-sharedir-perl liblwp-useragent-chicaching-perl libterm-readline-gnu-perl libjson-xs-perl libunicode-linebreak-perl libtext-charwidth-perl`

**Comment by trizen** [comment]
Thanks! One minor correction:

`liblwp-useragent-chicaching-perl` ([LWP::UserAgent::CHICaching](https://metacpan.org/pod/LWP::UserAgent::CHICaching)) is not needed.

Notice that it is different from [LWP::UserAgent::Cached](https://metacpan.org/pod/LWP::UserAgent::Cached) (what `pipe-viewer` uses, if available).

**Comment by pirate486743186** [comment]
yep, i can't find it then. Need more careful search, some other time.
